### PR TITLE
Removing Slack client and pushover notifications

### PR DESCRIPTION
**Pull request by drtyh2o**
Also, cleaned up imports to consistently `import logger from '@mds-core/mds-logger` and removed `async`

Finally, refactored, simplified, and cleaned up the mds-logger code that performs redaction.

This interesting changes are to this [file](https://github.com/lacuna-tech/mds-core/pull/226/files#diff-406a1eed236217488136e9620ea0ead1)

### CODE REVIEW

File: packages/mds-logger/index.ts [hunk 1]
```diff
@@ -14,231 +14,26 @@
...
-  return censoredObject
-}
-
-// eslint-disable-next-line @typescript-eslint/no-explicit-any
-function makeCensoredLogMsg(...msgs: any[]): any[] {
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  return makeCensoredLogMsgRecurse(msgs).map((item: any) =>
-    // never print out '[object Object]'
-    String(item) === '[object Object]' ? JSON.stringify(item) : item
+const redact = (args: unknown[]): string[] =>
```
Comment (janedotx) [inline packages/mds-logger/index.ts]: Do we not care about the redaction working on arbitrarily nested objects? This, for instance, fails if an array of objects containing GPS information is logged.

Comment (drtyh2o) [reply packages/mds-logger/index.ts]: Why would you say it fails? Did you try it? All tests pass, plus:

```ts
logger.info([
  { gps: { lat: 0, lng: 0 } },
  {
    points: [
      { lat: 0, lng: 0 },
      { lat: 1, lng: 1 },
      { lat: 2, lng: 2 }
    ]
  }
])
```
Produces:
`INFO [{"gps":{"lat":"[REDACTED]","lng":"[REDACTED]"}},{"points":[{"lat":"[REDACTED]","lng":"[REDACTED]"},{"lat":"[REDACTED]","lng":"[REDACTED]"},{"lat":"[REDACTED]","lng":"[REDACTED]"}]}]`

Comment (avatarneil) [reply packages/mds-logger/index.ts]: This does not fail in that case:
```
INFO [{"device_id":"ec551174-f324-4251-bfed-28d9f3f473fc","gps":{"lat":"[REDACTED]","lng":"[REDACTED]","speed":0,"hdop":1,"heading":180},"charge":0.5,"timestamp":1555384091559,"recorded":1555384091836},{"device_id":"ec551174-f324-4251-bfed-28d9f3f473fc","gps":{"lat":"[REDACTED]","lng":"[REDACTED]","speed":100,"hdop":10,"heading":20},"charge":0.75,"timestamp":1555384090000,"recorded":1555384090000}]
```

Comment (janedotx) [reply packages/mds-logger/index.ts]: > Why would you say it fails? Did you try it? All tests pass, plus:
> 
> ```ts
> logger.info(
>   { gps: { lat: 0, lng: 0 } },
>   {
>     points: [
>       { lat: 0, lng: 0 },
>       { lat: 1, lng: 1 },
>       { lat: 2, lng: 2 },
>       {
>         more: [
>           { lat: 3, lng: 3 },
>           { lat: 4, lng: 4 }
>         ]
>       }
>     ]
>   }
> )
> ```
> 
> Produces:
> `INFO {"gps":{"lat":"[REDACTED]","lng":"[REDACTED]"}} {"points":[{"lat":"[REDACTED]","lng":"[REDACTED]"},{"lat":"[REDACTED]","lng":"[REDACTED]"},{"lat":"[REDACTED]","lng":"[REDACTED]"},{"more":[{"lat":"[REDACTED]","lng":"[REDACTED]"},{"lat":"[REDACTED]","lng":"[REDACTED]"}]}]}`

I tried passing in an array to one of the tests and saw a failure, but that was my mistake. You're right, it does work.

**Pull request merged**